docs/source/dev/profiling/profiling_index.rst

```diff
@@ -1,5 +1,6 @@
-Profiling vLLM 
-=================================
+==============
+Profiling vLLM
+==============
```

Comment:
I assume this is just a style choice?

It would be nice to enforce this sort of thing with a linter if there's a preference for some reason.

Reply:
I assumed it was best practice basing it off the example [here](https://devguide.python.org/documentation/markup/#sections), but it's optional. Either way, a linter seems like a good idea to enforce! I could contribute that.

Reply:
Yeah, don't worry about the linting here. It was more of a "thinking out loud for the future" comment